File: Assets/Bundles/Shared/Resources/TS/Util/Theme.lua
-- Compiled with unity-ts v2.1.0-75
local Theme
do
	Theme = setmetatable({}, {
		__tostring = function()
			return "Theme"
		end,
	})
	Theme.__index = Theme
	function Theme.new(...)
		local self = setmetatable({}, Theme)
		return self:constructor(...) or self
	end
	function Theme:constructor()
	end
	Theme.Green = Color.new(0.5, 1, 0.5, 1)
	Theme.Red = Color.new(1, 85 / 255, 85 / 255, 1)
	Theme.Blue = Color.new(0.6, 0.6, 1, 1)
	Theme.Yellow = Color.new(1, 1, 0.39)
	Theme.White = Color.new(1, 1, 1, 1)
	Theme.Black = Color.new(0, 0, 0, 0)
	Theme.Gray = Color.new(0.63, 0.63, 0.63)
	Theme.Aqua = Color.new(86 / 255, 255 / 255, 255 / 255)
end
return {
	Theme = Theme,
}
-- ----------------------------------
-- ----------------------------------

File: Assets/Bundles/@Easy/Core/Core~/src/Shared/Util/Theme.ts
import { ColorUtil } from "./ColorUtil";

export class Theme {
	public static green = new Color(0.5, 1, 0.5, 1);
	public static red = new Color(1, 85 / 255, 85 / 255, 1);
	public static blue = new Color(0.6, 0.6, 1, 1);
	public static yellow = new Color(1, 1, 0.39);
	public static white = new Color(1, 1, 1, 1);
	public static black = new Color(0, 0, 0, 0);
	public static gray = new Color(0.63, 0.63, 0.63);
	public static aqua = new Color(86 / 255, 255 / 255, 255 / 255);
	public static pink = ColorUtil.HexToColor("#FF55FF");

	public static teamColor = {
		Green: new Color(0.5, 1, 0.5, 1),
		Red: new Color(1, 85 / 255, 85 / 255, 1),
		Blue: new Color(0.6, 0.6, 1, 1),
		Yellow: new Color(1, 1, 0.39),
	};
}
